defaultState.ts

```diff
@@ -0,0 +1 @@
+export const defaultState = '<callout>💡 This page is a sample of what you can include in your Home app. Edit the    content to give your clients a personalized experience when they open up the client portal.</callout><h1>Welcome, {{client.givenName}}</h1><p>Welcome to BrandMages, a full-service marketing agency that helps businesses increase their brand awareness, attract new customers, and grow their bottom line. We specialize in crafting unique and effective marketing strategies that align with your business goals and help you stand out in a crowded marketplace.</p><p>Here are our Working Hours:</p><table><tbody><tr><th class="font-bold" colspan="1" rowspan="1"><p>Days</p></th><th class="font-bold" colspan="1" rowspan="1"><p>Hours (EST)</p></th></tr><tr><td colspan="1" rowspan="1"><p>M-F</p></td><td colspan="1" rowspan="1"><p>9AM - 6PM</p></td></tr><tr><td colspan="1" rowspan="1"><p>Sat</p></td><td colspan="1" rowspan="1"><p>10AM - 5PM</p></td></tr><tr><td colspan="1" rowspan="1"><p>Sun</p></td><td colspan="1" rowspan="1"><p>Closed</p></td></tr></tbody></table><h1>How to use the BrandMages portal</h1><h3>💬&nbsp;Messages</h3><p>Chat with the BrandMages team or ask questions — make sure to check the Helpdesk first for some FAQs!</p><h3>📂&nbsp;Files</h3><p>Use this tab to share files with the BrandMages team — you can upload any required documents and assets here. This is where we will store files for reference in the future.</p><h3>📜&nbsp;Contracts</h3><p>View and sign contracts shared by our team.</p><h3>📤&nbsp;Forms</h3><p>We’ve already shared our onboarding form with you. Please fill out the form and include all relevant information, including your brand guidelines and company details.</p><h3>💵&nbsp;Billing</h3><p>Here you will see active subscription and invoices once they’ve been assigned to you.</p><h3>🆘&nbsp;Helpdesk</h3><p>Access a collection of helpful articles we have put together.</p><h3>📊&nbsp;Analytics</h3><p>View custom analytics reports for your business.</p><h3>🌇&nbsp;Media Gallery</h3><p>See any visual and design work that the BrandMages team has completed.</p><h3>📈&nbsp;Project status</h3><p>Track the projects we’re working on for you.</p>'
```

Comment:
Can we store this html in proper structure? If we have to change anything in the future, then it would be difficult

Reply:
export const defaultState = `

<callout>💡 This page is a sample of what you can include in your Home app. Edit the content to give your clients a personalized experience when they open up the client portal.</callout>
<h1>Welcome, {{client.givenName}}</h1>
<p>Welcome to BrandMages, a full-service marketing agency that helps businesses increase their brand awareness, attract new customers, and grow their bottom line. We specialize in crafting unique and effective marketing strategies that align with your business goals and help you stand out in a crowded marketplace.</p>
<p>Here are our Working Hours:</p>
<table>
   <tbody>
      <tr>
         <th class="font-bold" colspan="1" rowspan="1">
            <p>Days</p>
         </th>
         <th class="font-bold" colspan="1" rowspan="1">
            <p>Hours (EST)</p>
         </th>
      </tr>
      <tr>
         <td colspan="1" rowspan="1">
            <p>M-F</p>
         </td>
         <td colspan="1" rowspan="1">
            <p>9AM - 6PM</p>
         </td>
      </tr>
      <tr>
         <td colspan="1" rowspan="1">
            <p>Sat</p>
         </td>
         <td colspan="1" rowspan="1">
            <p>10AM - 5PM</p>
         </td>
      </tr>
      <tr>
         <td colspan="1" rowspan="1">
            <p>Sun</p>
         </td>
         <td colspan="1" rowspan="1">
            <p>Closed</p>
         </td>
      </tr>
   </tbody>
</table>
<h1>How to use the BrandMages portal</h1>
<h3>💬&nbsp;Messages</h3>
<p>Chat with the BrandMages team or ask questions — make sure to check the Helpdesk first for some FAQs!</p>
<h3>📂&nbsp;Files</h3>
<p>Use this tab to share files with the BrandMages team — you can upload any required documents and assets here. This is where we will store files for reference in the future.</p>
<h3>📜&nbsp;Contracts</h3>
<p>View and sign contracts shared by our team.</p>
<h3>📤&nbsp;Forms</h3>
<p>We’ve already shared our onboarding form with you. Please fill out the form and include all relevant information, including your brand guidelines and company details.</p>
<h3>💵&nbsp;Billing</h3>
<p>Here you will see active subscription and invoices once they’ve been assigned to you.</p>
<h3>🆘&nbsp;Helpdesk</h3>
<p>Access a collection of helpful articles we have put together.</p>
<h3>📊&nbsp;Analytics</h3>
<p>View custom analytics reports for your business.</p>
<h3>🌇&nbsp;Media Gallery</h3>
<p>See any visual and design work that the BrandMages team has completed.</p>
<h3>📈&nbsp;Project status</h3>
<p>Track the projects we’re working on for you.</p>

`

Reply:
changes made in the next commit @sazanrjb